diff -durN xrick.orig/config xrick/config
--- xrick.orig/config	Mon Aug  6 21:06:13 2001
+++ xrick/config	Tue Sep 18 12:21:10 2001
@@ -38,10 +38,10 @@
 
 ROOTDIR=`pwd`
 
-CFLAGS=
+CFLAGS="$CFLAGS"
 LDFLAGS=
 
-CFLAGS="$CFLAGS -g -ansi -pedantic -Wall -W -O2" # add -g to debug
+CFLAGS="$CFLAGS -ansi -pedantic -Wall -W" # add -g to debug
 CFLAGS="$CFLAGS -I $ROOTDIR/include"
 CFLAGS="$CFLAGS -DVERSION=\\\"$VERSION\\\""
 
